core_v2/sources/readme.md

```diff
@@ -0,0 +1,100 @@
+# Aptos Naming Service
```

Comment:
rofl, this is not near the top of the diff... so I read all that code up until now without any guidance...